queries.sql

```diff
@@ -20,3 +20,96 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 -- Animals weighing from 10.4 to 17.3kg
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--TRANSACTION
+-- set species to unspecified
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+--Verify if the change was made
+SELECT * FROM animals;
+
+--rollback the change
+ROLLBACK;
+
+--verify if the changes was rolledback
+SELECT * FROM animals;
+
+--Update species column to digimon for all animals that have a name ending in 'mon'. 
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update species column to pokemon for all animals that don't have species already set.
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+--Verify if the change was made and persists after commit.
+SELECT * FROM animals;
+
+--delete all records in the animals table
+BEGIN;
+DELETE from animals;
+ROLLBACK;
+
+--verify if the changes was rolledback
+SELECT * FROM animals;
+
+--Delete all animals born after Jan 1st, 2022.
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
```

Comment:
- [OPTIONAL]  Kindly make sure you understand the requirements well. This should be `DELETE FROM animals WHERE  date_of_birth >= '2022/01/01' ;`. 👍🏻 I strongly recommend you make this. In the meantime, I'm going to approve this project since it's the only thing you missed. 💪🏻 😄